### PR TITLE
[CCAP-433] Adjust logic and flows for submission and document upload screens

### PR DESCRIPTION
**Pull request by enyia21**
#### 🔗 Jira ticket
CCAP-433

#### ✍️ Description
Change the logic of the `submit-complete` screen and to match Figma

#### 📷 Design reference
[Figma](https://www.figma.com/design/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Families-Full-Flow?node-id=11016-30028&t=PQgZayvwiDX0aqQf-4)

#### ✅ Completion tasks
- [ ] Remove the alternative version of the submit-complete screen for users without recommended documents 
- [ ] Update the submit-complete screen for all users to match the Figma designs with content from the content manager.
- [ ] Skip the doc-upload-recommended-docs screen if a user has no recommended documents

- [ ] Hide the accordion on the doc-upload-add-files screen if a user has no recommended documents
- [ ] This work can be implemented right away in the current flow and does not need to be hidden by a feature flag

#### 👍 Acceptance
- [ ] Added relevant tests
- [ ] Meets acceptance criteria

### CODE REVIEW

File: src/main/resources/messages.properties [hunk 1]
```diff
@@ -701,7 +701,7 @@ unearned-income-referral-services.housing-support=Housing support and homelessne
 unearned-income-referral-services.disability-support=Services for people living with a physical or mental disability
 
 #
-submit-intro.step=Step 5 of 5
+submit-intro.step=Step 5 of 6
```
Comment (analoo) [inline src/main/resources/messages.properties:704]: @enyia21 - Section 6 is actually behind a feature flag so I do think this piece should be conditional on the feature flag

Comment (analoo) [reply src/main/resources/messages.properties:704]: like 5 of 5 if feature flags are off, 5 of 6 if feature flags are on.

File: src/test/java/org/ilgcc/app/journeys/DocumentUploadConditionalLogicJourneyTest.java [hunk 2]
```diff
@@ -3,12 +3,11 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import java.util.List;
 import org.ilgcc.app.utils.AbstractBasePageTest;
-import org.ilgcc.app.utils.ChildCareProvider;
 import org.junit.jupiter.api.Test;
 public class DocumentUploadConditionalLogicJourneyTest extends AbstractBasePageTest {
     boolean hasPartner = false;
     @Test
-    void shouldSkipDocUploadPromptIfNoDocumentsAreRequiredForParentOrSpouse(){
+    void shouldSkipDocUploadPromptIfNoDocumentsAreRequiredForParentOrSpouseAndShouldHideRecommendedDocumentsAccordionOnTheDocuUploadAddFilesScreen(){
```
Comment (analoo) [inline src/test/java/org/ilgcc/app/journeys/DocumentUploadConditionalLogicJourneyTest.java:10]:
```suggestion
    void SkipsRecommendedDocumentsScreenIfNoneAreRequiredFor(){
```

File: src/test/java/org/ilgcc/app/journeys/DocumentUploadConditionalLogicJourneyTest.java [hunk 3]
```diff
@@ -24,13 +23,14 @@ void shouldSkipDocUploadPromptIfNoDocumentsAreRequiredForParentOrSpouse(){
...
-    assertThat(testPage.getTitle()).isEqualTo(getEnMessageWithParams("submit-confirmation.title", List.of(ChildCareProvider.OPEN_SESAME.getDisplayName()).toArray()));
+    testPage.clickButton(getEnMessage("submit-complete.yes-add-document-now"));
 
+    //doc-upload-add-files
+    assertThat(testPage.getTitle()).isEqualTo(getEnMessage("doc-upload-add-files.title"));
+    assertThat(testPage.elementDoesNotExistById("controlId")).isTrue();
     }
     @Test
-    void skipNotSkipDocUploadRecommendedDocsScreenIfOneDocumentIsRequiredForParent(){
+    void skipNotSkipDocUploadRecommendedDocsScreenIfOneDocumentIsRequiredAndShouldDisplayRecommendedDocumentsAccordionOnTheDocUploadAddFilesScreen(){
```
Comment (analoo) [inline src/test/java/org/ilgcc/app/journeys/DocumentUploadConditionalLogicJourneyTest.java:33]:
```suggestion
    void DisplaysRecommendedDocumentsScreenIfDocsAreRequired()
```

File: src/main/resources/messages.properties (hunk 1 above)
Comment (enyia21) [reply src/main/resources/messages.properties:704]: Yeah this is actually part of another ticket.  I'm going to remove these changes from the PR

File: src/test/java/org/ilgcc/app/journeys/DocumentUploadConditionalLogicJourneyTest.java [hunk 4]
```diff
@@ -3,12 +3,11 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import java.util.List;
 import org.ilgcc.app.utils.AbstractBasePageTest;
-import org.ilgcc.app.utils.ChildCareProvider;
 import org.junit.jupiter.api.Test;
 public class DocumentUploadConditionalLogicJourneyTest extends AbstractBasePageTest {
     boolean hasPartner = false;
     @Test
-    void shouldSkipDocUploadPromptIfNoDocumentsAreRequiredForParentOrSpouse(){
+    void SkipsRecommendedDocumentsScreenIfNoneAreRequiredFor(){
```
Comment (analoo) [inline src/test/java/org/ilgcc/app/journeys/DocumentUploadConditionalLogicJourneyTest.java:10]:
```suggestion
    void SkipsRecommendedDocumentsScreenIfNoneAreRequired(){
```

**Pull request merged**